src/lib/data/audio.ts

```diff
@@ -274,16 +281,28 @@ export async function getAudioSourceInfo(item: {
     };
 }
 
-//todo implement selectable text that can play audio
-
-// export function onClickSound(e: any) {
-//     let target = e.target;
-
-//     if (isSelectableText(target)) {
-//         const start = parseFloat(target.start);
-//         this.audio.currentTime = start;
-//         this.audio.play();
-//     } else {
-
-//     }
-// }
+// changes audio to the verse number clicked on
+export function verseClickSoundChange(verseId) {
```

Comment:
This function is similar to `seek(position)`, right? You just have to find the position from the timing files.

Please change this to:
1. `seekToVerse(verseId)`
2. return if`!hasAudioPlayed()`
3. find the new position from the timing info.
4. call `seek(position)` so this code is not repeated (which will do the pause/play logic).